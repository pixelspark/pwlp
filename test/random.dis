0000.	e0	USER	get_length
0001.	e5	USER	random_int
0002.	e0	USER	get_length
0003.	20	PEEKB	0
0004.	11	PUSHB	[01]
0006.	81	BINARY	SUB
0007.	10	PUSHB	0
0008.	74	UNARY	SHL8
0009.	86	BINARY	OR
0010.	10	PUSHB	0
0011.	74	UNARY	SHL8
0012.	74	UNARY	SHL8
0013.	86	BINARY	OR
0014.	10	PUSHB	0
0015.	74	UNARY	SHL8
0016.	74	UNARY	SHL8
0017.	74	UNARY	SHL8
0018.	86	BINARY	OR
0019.	e3	USER	set_pixel
0020.	01	POP	1
0021.	20	PEEKB	0
0022.	22	PEEKB	2
0023.	8c	BINARY	EQ
0024.	50	JZ	to 45
0027.	21	PEEKB	1
0028.	11	PUSHB	[ff]
0030.	74	UNARY	SHL8
0031.	86	BINARY	OR
0032.	11	PUSHB	[ff]
0034.	74	UNARY	SHL8
0035.	74	UNARY	SHL8
0036.	86	BINARY	OR
0037.	11	PUSHB	[ff]
0039.	74	UNARY	SHL8
0040.	74	UNARY	SHL8
0041.	74	UNARY	SHL8
0042.	86	BINARY	OR
0043.	e3	USER	set_pixel
0044.	01	POP	1
0045.	01	POP	1
0046.	71	UNARY	DEC
0047.	60	JNZ	to 3
0050.	01	POP	1
0051.	e4	USER	blit
0052.	e0	USER	get_length
0053.	20	PEEKB	0
0054.	11	PUSHB	[01]
0056.	81	BINARY	SUB
0057.	10	PUSHB	0
0058.	74	UNARY	SHL8
0059.	86	BINARY	OR
0060.	10	PUSHB	0
0061.	74	UNARY	SHL8
0062.	74	UNARY	SHL8
0063.	86	BINARY	OR
0064.	10	PUSHB	0
0065.	74	UNARY	SHL8
0066.	74	UNARY	SHL8
0067.	74	UNARY	SHL8
0068.	86	BINARY	OR
0069.	e3	USER	set_pixel
0070.	01	POP	1
0071.	71	UNARY	DEC
0072.	60	JNZ	to 53
0075.	01	POP	1
0076.	e4	USER	blit
0077.	01	POP	1
0078.	40	JMP	to 0

