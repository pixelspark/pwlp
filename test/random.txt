
loop {
	lucky = random(get_length);
	for(n=get_length) {
		set_pixel(n-1, 0, 0, 0);
		if(n == lucky) {
			set_pixel(n, 255, 255, 255);
		}
	};
	blit;
	for(n=get_length) {
		set_pixel(n-1, 0, 0, 0);
	};
	blit;
	yield;
}