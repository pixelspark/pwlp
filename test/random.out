000000 ffffff 000000 000000 000000 000000 000000 000000 000000 000000 
000000 000000 000000 000000 000000 000000 000000 000000 000000 000000 
000000 000000 000000 000000 000000 000000 000000 ffffff 000000 000000 
000000 000000 000000 000000 000000 000000 000000 000000 000000 000000 
000000 ffffff 000000 000000 000000 000000 000000 000000 000000 000000 
000000 000000 000000 000000 000000 000000 000000 000000 000000 000000 
000000 000000 000000 000000 000000 000000 000000 000000 ffffff 000000 
000000 000000 000000 000000 000000 000000 000000 000000 000000 000000 
000000 000000 000000 000000 000000 ffffff 000000 000000 000000 000000 
000000 000000 000000 000000 000000 000000 000000 000000 000000 000000 
000000 000000 ffffff 000000 000000 000000 000000 000000 000000 000000 
